engine/src/main/java/org/hibernate/validator/messageinterpolation/AbstractMessageInterpolator.java

```diff
@@ -166,61 +175,75 @@ public AbstractMessageInterpolator(ResourceBundleLocator userResourceBundleLocat
 	public AbstractMessageInterpolator(ResourceBundleLocator userResourceBundleLocator,
 			ResourceBundleLocator contributorResourceBundleLocator,
 			boolean cacheMessages) {
-		this( userResourceBundleLocator, contributorResourceBundleLocator, Collections.emptySet(), cacheMessages );
+		this( userResourceBundleLocator, contributorResourceBundleLocator, Collections.emptySet(), Locale.getDefault(), cacheMessages );
 	}
 
 	/**
 	 * {@code MessageInterpolator} using the default resource bundle locators.
 	 *
-	 * @param localesToInitialize The set of locales to initialize at bootstrap.
+	 * @param localesToInitialize the set of locales to initialize at bootstrap
+	 * @param defaultLocale the default locale
 	 *
-	 * @since 6.1
+	 * @since 6.1.1
 	 */
-	public AbstractMessageInterpolator(Set<Locale> localesToInitialize) {
```

Comment:
Yeah, it should have been made `@Incubating` from the beginning but I missed it and it's only used for the `PredefinedScope` stuff and I thought it would be better to drop it rather than stacking all these constructors.

The new ones are properly marked as `@Incubating`.